coordinator/cluster/cluster.go

```diff
@@ -91,9 +108,9 @@ func (c *LoadAgentCluster) Shutdown() {
 	var wg sync.WaitGroup
 	wg.Add(len(c.agents))
 	for _, ag := range c.agents {
-		go func(ag *agent.LoadAgent) {
+		go func(ag *client.Agent) {
 			defer wg.Done()
-			if err := ag.Stop(); err != nil {
+			if _, err := ag.Stop(); err != nil {
```

Comment:
```suggestion
			if _, err := ag.Destroy(); err != nil {
```

The previous behaviour was actually to destroy the agents. Not sure why it ended up being called `Stop()`.
This actually turned out to be useful and uncovered [MM-26694](https://mattermost.atlassian.net/browse/MM-26694) during testing.